tests/testthat/test-fcm_methods.R

```diff
@@ -135,7 +135,8 @@ test_that("longer selection than longer than features that exist (related to #44
     feat <- c("b", "c", "d", "e", "f", "g")
     # bugs in C++ needs repeated tests
     expect_message(fcm_select(fcmt_test2, feat, verbose = TRUE),
-                   "kept 4 features")
+                   "dfm_keep() changed from 5 features (5 documents) to 4 features (5 documents)",
```

Comment:
Should be `fcm_keep()` or even the specific function call - here it refers to `keep` when the parent function was `select`.

Reply:
I think we need a `message_fcm()` function.